07_cmd_lab/cmdlab.txt

```diff
@@ -1,15 +1,54 @@
+Hi Ynon,
+Below you finf my answers. You don't mind english, right? (hebrew and english are not native for me, but I got used to write in english)
 Intro To Command Line Lab.
 Questions online: https://www.tocode.co.il/bundles/unix1/lessons/cmdlab
 
 1. Login to your UNIX account. How do you find:
-  - your current shell?
-  - your default shell?
-  - a list of all shells currently installed on the machine?
 
-עברית שפה קשה. בעקרון המעטפת הנוכחית שלי היא bash
+  - your current shell?  $ echo $0
 
-2. View the file /etc/passwd on your machine with the command "cat /etc/passwd". Then read about the file in "man 5 passwd". What information is held in the file? What are the fields in each line?
+  - your default shell?  $ echo $SHELL
 
-3. In Unix, the /etc folder holds configuration files. Display all files in "/etc" folder ordered by file size (Hint: search in "man ls")
+  - a list of all shells currently installed on the machine?  $ cat /etc/shells
 
-4. Print current date/time formatted as: "Sun 21/03/2010, 14:30". Hint: read "man date" and search for the correct format string.
+
+
+2. View the file /etc/passwd on your machine with the command "cat /etc/passwd". Then read about the file in "man 5 passwd". a. What information is held in the file? b. What are the fields in each line?
+
+a. In short: info about /etc/passwd.
+
+DESCRIPTION 
+/etc/passwd contains one line for each user account, with seven fields delimited by colons (":").
+
+*By the way, if the last field id "optional user command interpreter", what does it mean "nologin". Is "nologin" also interpreter, like bash?
```

Comment:
Absolutely. It's a small utility that prints something like `This account is currently not available.` and quits. It helps to prevent logins in system accounts 
(we'll talk about them later in the course)